homeassistant/components/google_travel_time/config_flow.py

```diff
@@ -49,6 +51,15 @@
 )
 from .helpers import InvalidApiKeyException, UnknownException, validate_config_entry
 
+CONFIG_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_NAME, default=DEFAULT_NAME): cv.string,
```

Comment:
Don't think we should allow changing the name on reconfigure as they can change it on the config entry directly.